### PR TITLE
Expose logger factory 🏭

### PR DESCRIPTION
**Pull request by keirog**
Allows self-contained use of the addContext method per component.

Use-case: If you have an analytics component in your app, you might want
to use the [logger.addContext](https://github.com/financial-times/n-logger#addcontextmeta) method to attribute `logSource: 'analytics'` to all logs that come from that component. This is much easier to achieve if each component can have their own instance of the logger ✌️.

 🐿 v2.11.0

### CODE REVIEW

**Comment by sjparkinson** [comment]
Mind updating the documentation/readme while you're in the area?

Good to have down some example code on how this might be used.

**Comment by AbdullahAli** [comment]
Thanks @sjparkinson, we have added the following [docs](https://github.com/Financial-Times/n-logger/pull/86/commits/e1064ae211b9e1cba379c691a00e4e3233de4a38)

Please let me know if you are happy to merge this

File: README.md [hunk 1]
```diff
@@ -25,6 +25,24 @@ If using CommonJS modules
 
     const logger = require('@financial-times/n-logger').default;
 
+You may also have different instances of the logger in the same project.  This is useful if you wanted to have different [context](#addcontextmeta) per logger:
+
+```javascript
+  import { getLogger } from '@financial-times/n-logger';
```
Comment (sjparkinson) [inline README.md]: Given `@financial-times/n-logger` is used for server-side code, better to use...

We're trying to remove Babel from projects that don't need it.

```suggestion
const logger = require('@financial-times/n-logger');
```

Comment (AbdullahAli) [reply README.md]: Good shout, I have corrected it here: https://github.com/Financial-Times/n-logger/pull/86/commits/77514c7a778a07ea45b484348aabb129f694ea27

**Comment by edds** [comment]
This PR is being closed due to inactivity as part of a Pull Request Bankruptcy. If you feel it’s still useful or relevant please feel free to re-open it.

**Pull request closed without merging**